### PR TITLE
Check "edit_motech" permission

### PR DESCRIPTION
**Pull request by kaapstorm**
Allow users with the "edit_motech" permission to configure OpenMRS and DHIS2 configuration.

code buddy @gbova cc @esoergel

### CODE REVIEW

File: corehq/motech/repeaters/views/repeaters.py [hunk 1]
```diff
@@ -64,6 +64,7 @@ def page_context(self):
             'repeaters': self.repeaters,
             'pending_record_count': RepeatRecord.count(self.domain),
             'gefingerpoken': self.request.couch_user.is_superuser or
+                             self.request.couch_user.has_permission(self.domain, 'edit_motech') or
```
Comment (kaapstorm) [inline corehq/motech/repeaters/views/repeaters.py]: The template shows the Data Forwarder's "Configure" button if "gefingerpoken" is set.

File: corehq/tabs/tabclasses.py [hunk 2]
```diff
@@ -1437,7 +1437,7 @@ def sidebar_items(self):
         if user_is_admin:
             items.append((_('Project Administration'), _get_administration_section(self.domain)))
 
-        if self.couch_user.can_edit_motech:
+        if self.couch_user.has_permission(self.domain, 'edit_motech'):
```
Comment (kaapstorm) [inline corehq/tabs/tabclasses.py]: I noticed that `self.couch_user.can_foo` is truthy for any "foo". That doesn't make sense to me. I'll follow up.

File: corehq/motech/repeaters/views/repeaters.py [hunk 3]
```diff
@@ -63,8 +63,11 @@ def page_context(self):
         return {
             'repeaters': self.repeaters,
             'pending_record_count': RepeatRecord.count(self.domain),
-            'gefingerpoken': self.request.couch_user.is_superuser or
-                             toggles.IS_CONTRACTOR.enabled(self.request.couch_user.username)
+            'gefingerpoken': (
```
Comment (kaapstorm) [inline corehq/motech/repeaters/views/repeaters.py]: The template shows the Data Forwarder's "Configure" button if "[gefingerpoken][1]" is set.

  [1]: https://en.wikipedia.org/wiki/Blinkenlights

Comment (esoergel) [reply corehq/motech/repeaters/views/repeaters.py]: haha, that's cute.  An in-line comment would help make this readable if you're committed to the reference.

**Review by esoergel — APPROVED** [review]

**Review by gbova — APPROVED** [review]

File: corehq/tabs/tabclasses.py (hunk 2 above)
Comment (kaapstorm) [reply corehq/tabs/tabclasses.py]: Good to know! I feel a bit ambivalent about how opaque this is, but the result is more readable, so I'll go with it.

**Pull request merged**